src/components/GradientItem.jsx

```diff
@@ -1,23 +1,20 @@
 import React from "react";
 
-const GradientItem = props => {
+const GradientItem = (props) => {
+  const { inputValueSecond, inputValueFirst, isSubmitted } = props;
```

Comment:
Теперь тут тебе нужен только один проп
```suggestion
const GradientItem = ({ colorSet }) => {
```